flexmeasures/api/v3_0/assets.py

```diff
@@ -665,6 +665,7 @@ def patch(self, asset_data: dict, id: int, db_asset: GenericAsset):
                         latitude: 11.1
                         longitude: 99.9
                         account_id: 1
+                        external_id: ""
```

Comment:
Given that the db column is nullable, would we expect an empty string?

Reply:
Probably None would come back then, you are right.